### ISSUE TITLE
Add an option to the editor to specifically set RTL

### ISSUE THREAD

**Issue by AriaShishegaran** [wix/ricos#2139]
while Ricos supports RTL by detecting whether the sentence is beginning with an ltr or RTL language when writing articles that might include a certain character or word that is English or in any other ltr language the editor thinks that it should change the direction which causes massive inconsistencies.
![image](https://user-images.githubusercontent.com/15879200/107694412-97236080-6cc4-11eb-886e-758c62e2a07e.png)

please address this problem like many other rich editors by allowing the user to choose which direction text should be and leave the rest at that.

**Comment by mohessaid** [comment]
I think the auto-detection feature is what makes Draftjs and Ricos great. I don't know about Farsi but in your case, our case starting a sentence with an English word or say a word from an LTR language or foreign language, in general, is not very common. Most of the time we start it with Arabic words like "تطبيق Clubhouse" or write it in Arabic letters "كلوب هاوس (Clubhouse)" and put the English version between parentheses. Of course, everyone has their preferences.

I would like to keep the auto-detection feature and maybe add what you requested as an option to force writing direction.

**Comment by AriaShishegaran** [comment]
@mohessaid I totally agree, regarding the auto-detect feature, but considering the direction, websites might follow, maybe a person wants to write some article that is technical and yet not veery suitable for translation and therefore deciding on keeping the title as-is and continue elaborating as the rest of the article follows, i believe just like MS Word has the option to force the direction is great to have just in case.

looking forward to seeing this feature in the upcoming releases.

**Comment by mohessaid** [comment]
@AriaShishegaran Quiljs started with this feature. You can add the language direction controller or module to the toolbar so you can enforce a specific part of your text in a particular direction. They also have the option to initialize your editor with either LTR or RTL from the get-go. Then, they added the auto-detection feature. However, it's not as good as Ricos yet.
